ROADMAP.md

```diff
@@ -1,30 +1,32 @@
 Please take the timelines & dates as proposals and goals. Priorities and requirements change based on community feedback, roadblocks encountered, community contributions, etc. If you depend on a specific item, we encourage you to attend community meetings to get updated status information, or help us deliver that feature by contributing to Carvel.
 
-|Theme|Description|Timeline|
-|---|---|---|
-| Package Author Improvements | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents) Experimental: Configuration authors can specify the valid range or format of the data values. | Late July |
-| Provide Stability | **[kapp-controller]** [Surface namespaces for deployed resources](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/535) | Early August |
-| Provide Stability | **[kapp-controller]** Version safeguards: To provide safeguards and visibility for kapp-controller and kubernetes versions. | Early August |
-| Package Author Improvements | **[kctrl]** kctrl commands for package authors. | Aug 2022 |
-| Package Author Improvements | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents) GA: Configuration authors can specify the valid range or format of the data values. | Late August |
-| Provide Stability | **[kapp-controller]** Minimize registry communications: To provide a more performant experience for users leveraging large PackageRepos. | Late August |
-
+`Last Updated: Aug 2022`
+|Theme|Feature|Stage|Timeline|
+|---|---|---|---|
+| Package Author Experience | **[kctrl]** [kctrl commands for package authors.](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/632): CLI-based Package Author commands to enable  Package Authors easily create a Carvel package of their software | Build | Q3 (Jul-Sep) |
+| Package Author Experience | **[kctrl]** [kctrl dev deploy.](https://github.com/vmware-tanzu/carvel-kapp-controller/issues/2): Enable quick iteration on Package CRs and App CRs using kapp-controller CLI for local development | Build | Q3 (Jul-Sep) |
+| Package Author Experience | **[ytt]** [Schema Validations](https://hackmd.io/pODV3wzbT56MbQTxbQOOKQ#Part-7-Validating-Documents): Configuration authors can specify the valid range or format of the data values. || Q3 (Jul - Sep) |
+| Package Author Experience | **[carvel]** Carvel supports the ability to sign and verify assets (such as images, bundles, pkg/pkgr). |Awaiting Proposal| TBD |
+| Easy to Get Started | **[ytt]** [Guides & Examples](https://github.com/vmware-tanzu/carvel-ytt/issues/314): Provide more guides and examples so that ytt is easy to get started with and details how it can be incorporate in different workflows. [Epic](https://app.zenhub.com/workspaces/carvel-backlog-6013063a24147d0011410709/board?epics=173207060_314&filterLogic=any&repos=173207060) |Awaiting Proposal| TBD |
```

Comment:
I'm not sure why ytt Guides & Examples was added. I'd remove it as it currently isn't planned.

Reply:
it is from the existing Roadmap. (https://github.com/vmware-tanzu/carvel/blob/roadmap-strucutre-and-stages/ROADMAP.md). Want me to remove it?

Reply:
removed

Reply:
Thanks! To clarify, the existing roadmap doesn't have this item: https://github.com/vmware-tanzu/carvel/blob/develop/ROADMAP.md ([durable link](https://github.com/vmware-tanzu/carvel/blob/7285c3c29ac99414efb2582fcb31592f2b00bc07/ROADMAP.md)).